test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -1,207 +1,47 @@
 require "integration_test_helper"
 require "gds_api/test_helpers/mapit"
+require_relative "../support/coronavirus_local_restrictions_helpers"
 
 class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
   include GdsApi::TestHelpers::Mapit
   include GdsApi::TestHelpers::ContentStore
```

Comment:
I think you can remove both of these too.

Reply:
thanks, have removed.